canonical/ubuntu-pro-client
# tests: parametrized clouds

no-lp no-gh no-jira

## Test Steps

Make sure the existing `behave-$machine_type-$release` suites still execute the same set of tests, including those modified.

Try out these new commands that work:
```
tox -e behave-any -- features/apt_messages.feature:639
tox -e behave-any -- features/apt_messages.feature:640
tox -e behave-any -- -n "Cloud and series-specific URLs"
```

## Checklist

 - [n/a] I have updated or added any unit tests accordingly
 - [n/a] I have updated or added any documentation accordingly

## Does this PR require extra reviews?
